docs/source/en/_toctree.yml

```
@@ -33,7 +33,9 @@
     - local: converting_tensorflow_models
       title: Converting from TensorFlow checkpoints
     - local: serialization
-      title: Export 🤗 Transformers models
+      title: ONNX
+    - local: torchscript
+      title: TorchScript
```

We should keep the "Export models" here, for instance:
```suggestion
      title: Exporting models to ONNX
    - local: torchscript
      title: Exporting models with TorchScript
```